scripts/install.sh

```diff
@@ -32,7 +32,7 @@ function init() {
         API_SERVICE_ADDRESS="$LOAD_BALANCER_ADDRESS:$LOAD_BALANCER_PORT"
     fi
 
-    OLD_LOAD_BALANCER_ADDRESS=$(kubernetes_load_balancer_address)
+    local oldLoadBalancerAddress=$(kubernetes_load_balancer_address)
```

Comment:
Read in https://github.com/replicatedhq/kURL/pull/785/files#diff-e076e43cb817ecc8b3c2ed18cd58baa50544ecc66293ed6e61129c43ca6a09ccR139